integrations/micrometer/micrometer/src/main/java/module-info.java

```diff
@@ -32,17 +32,17 @@
 
     requires java.logging;
 
-    requires static jakarta.annotation;
+    requires jakarta.annotation;
 
     requires io.helidon.config;
-    requires io.helidon.common.http;
-    requires io.helidon.nima.servicecommon;
+    requires transitive io.helidon.common.http;
```

Comment:
this should not be `transitive`, as the API does not expose any class from common HTTP

Reply:
As far as I can see, the following types are used in public API:
- `FeatureSupport`  - implemented through `HelidonFeatureSupport`, relevant for the capability to be registered with WebServer, so I would not add it as a `transitive` (see below, we actually need to add this as transitive due to another used type)
- `Builder` from common
- `Config` from common-config
- `MeterRegistry` from micrometer
- `CrossOriginConfig` from cors
- `ResetServiceSettings` from `servicecommon` (used in builder, so must be transitive)
- `Function<ServerRequest, Optional<Handler>>` - OK, this is the problematic one - this would require transitive dependency on webserver. Please check with @tjquinno if this method is needed and should be considered user accessible public API. If so, it should be `transitive`